hack/build/common.sh

```diff
@@ -21,3 +21,5 @@ BUILD_DIR=${CDI_DIR}/hack/build
 MANIFEST_TEMPLATE_DIR=${CDI_DIR}/manifests/templates
 MANIFEST_GENERATED_DIR=${CDI_DIR}/manifests/generated
 SOURCE_DIRS="pkg tests tools"
+APIDOCS_OUT_DIR=${OUT_DIR}/apidocs
+PYTHON_CLIENT_OUT_DIR=${OUT_DIR}/cdi-client-python
```

Comment:
We still need this variable since we are not doing the python client anymore?

Reply:
good catch